apps/mobile/src/components/TabNavivation/TabBar.tsx

```diff
@@ -1,14 +1,37 @@
 import type { FC } from 'react';
-import type { ViewStyle } from 'react-native';
+import type { ImageSourcePropType, ViewStyle } from 'react-native';
 import { StyleSheet, View } from 'react-native';
 import type { BottomTabBarProps } from '@react-navigation/bottom-tabs';
+import { useDrawerStatus } from '@react-navigation/drawer';
+import type { DrawerNavigationHelpers } from '@react-navigation/drawer/lib/typescript/src/types';
 import type { Route } from '@react-navigation/native';
+import { appActions, appState } from 'state/app';
+import { nativeAsset } from 'utils/config';
 
 import NavigationItem from './Item';
-export const BottomNavigationTabBar: FC<BottomTabBarProps> = ({
+
+const getIconImage = (routeName: string): ImageSourcePropType | undefined => {
+	switch (routeName) {
+		case 'Profile':
+			return { uri: appState.profile.profileImage };
+		case 'Explore':
+			return nativeAsset.tabBar.explore;
+		case 'OurProject':
+			return nativeAsset.tabBar.walless;
+		default:
+			break;
```

Comment:
I don't think we have the same convention on Extension yet -- will this work together with Extension implementation?

Reply:
I don't think this one is related to the extension, it is the Bottom Tab icons which only used on mobile, I guess?